### ISSUE TITLE
Bug when unlocking via fingerprint sensor

### ISSUE THREAD

**Issue by fallsdevil** [jamie-mh/AuthenticatorPro#974]
Always when you open the app on the first try, it closes unexpectedly and shows a message written "fingerprint operation canceled".

**To Reproduce**
1. open the app
2. position my fingerprint, seconds later
3. the application closes
4. pop-up with error message

**Expected behavior**
That the app unlocked normally on the first attempt, and not just on the second after the strange error on the first time.

**App Version**
1.22.0

**Additional context**
Image with the error message, it is in Portuguese as it is my device language. Also, I downloaded the app from the Play Store
![ResizedImage_2023-10-12_17-43-45_3383](https://github.com/jamie-mh/AuthenticatorPro/assets/142761950/f5c90f06-8e09-4a5b-a403-f2369f5658bd)

**Comment by HigH-HawK** [comment]
Hey @fallsdevil 

Can you please add your device info (phone / tablet make/model and android version)

Might make it easier to reproduce and find the error

**Comment by fallsdevil** [comment]
Of course, I was using the Motorola Moto E22 smartphone running Android 12.
I'm currently using the Galaxy A10s Android 11 which has the same bug.

**Comment by HigH-HawK** [comment]
It seems that someone else has the same issue in #851 (at the end)

**Comment by fallsdevil** [comment]
I upgraded to the latest version of the play store and it's having the same problem, so the solution would be to disable the digital sensor unlock?

**Comment by jamie-mh** [comment]
I am not able to reproduce this on any of my devices or the emulator. You probably have an app on your device that interferes with the fingerprint scanning. Do you have an app that uses an accessibility service such as a password manager for instance installed?

Thanks

**Comment by fallsdevil** [comment]
Yes, I use bitwarden and maybe macrodroid has access to this accessibility, I will try to remove it to test.

**Comment by fallsdevil** [comment]
I disabled all applications with this accessibility and now the app no longer opens, it always closes a few seconds after opening every time...

**Comment by ouzowtf** [comment]
I have the same problem on my Pixel 4a 5G with Android 14 since a few months.
It works normally on the second try, but always produces the described error message on the first try.

**Comment by NebularNerd** [comment]
It seems to be a weird glitch that cannot be traced (see #851), both myself and @jamie-mh have Samsung S22's, mine demonstrates the bug (and it did on a previous phone), Jamie's does not. 

As suggested before, maybe a debug version that outputs its own logs might help.

**Comment by PingoLeon** [comment]
I have this problem too, it happens after you close the app and force shut it down. When you try to re-open it, there is this weird behavior where it tries to start the app and then immediately after quit and display the same message, saying the operation of fingerprint auth was canceled by user.. (couldn't read the end of the message) 
When you open right after, it works as expected.

Oneplus 9 - AOSP 14
Please note that I have a password manager if this could cause the problem.

**Comment by jamie-mh** [comment]
Is this still an issue in the latest version?

**Comment by NebularNerd** [comment]
It looks to be ok on 1.2.5, hopefully this bug is now quashed. 😎

Samsung S22 Ultra OneUI 7

**Comment by ouzowtf** [comment]
Seems to be fixed already.
